src/Orleans/Streams/Providers/IStreamProviderRuntime.cs

```diff
@@ -99,6 +99,9 @@ public class PersistentStreamProviderConfig
         public const string SILO_MATURITY_PERIOD = "SiloMaturityPeriod";
         public static readonly TimeSpan DEFAULT_SILO_MATURITY_PERIOD = TimeSpan.FromMinutes(2);
 
+        //optional param. user need to set this when they use Custom StreamQueueBalancer
+        public const string QUEUE_BALANCER_FACTORY_NAME = nameof(QueueBalancerFactoryName);
+        public string QueueBalancerFactoryName { get; set; }
```

Comment:
User needs to firstly set QueueBalancerFactoryName and set BalancerType to be CustomBalancer, secondly inject their `IStreamQueueBalancerFactory` with the set name to DI to be able to use this feature.